co.yaml

```diff
@@ -0,0 +1,512 @@
+# Colombian holiday definitions for the Ruby Holiday gem.
+#
+# Updated: 2018-06-01.
+#
+# Sources:
+# - https://en.wikipedia.org/wiki/Public_holidays_in_Colombia
```

Comment:
Thank you for teaching sites. 😃 
Following users (including me) can know about Colombia holiday.

Reply:
Sure, my pleasure! :blush: